README.md

```diff
@@ -19,6 +34,21 @@ npm run generate
 
 Open in a browser **public/index.html**
 
+## Running the tests
+```
+TODO: Add an automated testing framework
+```
+
+## Contributing
```

Comment:
PLease add informations about the commits, we are using commitizen and sematic release, the commits should be doing using:
npm run commit

Note: The ghooks are still not properly working, but in theory in the future a user can not do commit without use this comman or follow the standars.